spec/controllers/instrument_relays_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require "rails_helper"
```

Comment:
I should have mentioned this up front - we are moving away from controller specs where we can.  Do you mind re-writing this as a system spec instead?  For a simple example check out `access_list_tab_spec.rb`, or any other spec in the system directory that has a name ending like `...controller_spec.rb`.